.github/workflows/create-demo-clusters.yml

```diff
@@ -279,28 +308,54 @@ jobs:
-          kubectl -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+      - name: Post to Slack
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ needs.properties.outputs.slack-channel }}
+          payload: >-
+            {
+              "text": "Long-running cluster created. Setup your local access with `scripts/release-tools/setup-central-access.sh | bash -s -- ${{ steps.launch-central.outputs.cluster_name }}`",
 
-          kubectl -n stackrox set env deploy/central MUTEX_WATCHDOG_TIMEOUT_SECS=0
-          kubectl -n stackrox patch deploy/central -p '{"spec":{"template":{"spec":{"containers":[{"name":"central","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":tada: *Long-running cluster `${{ steps.launch-central.outputs.cluster_name }}` created for ${{ github.event.inputs.version }} milestone of <${{ github.server_url }}/${{ github.repository }}|${{ github.repository }}>.*"
+                  }
+                },
+                {
+                  "type": "divider"
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":arrow_right: Setup your local access to Central by running:\n```curl -L https://raw.githubusercontent.com/${{ github.repository }}/${{ github.ref_name }}/scripts/release-tools/setup-central-access.sh | bash -s -- ${{ steps.launch-central.outputs.cluster_name }}```"
+                  }
+                }
+              ]
+            }
 
-          ROX_ADMIN_PASSWORD=$(cat deploy/k8s/central-deploy/password)
-          echo "::add-mask::$ROX_ADMIN_PASSWORD"
-          CENTRAL_IP=$(kubectl -n stackrox get svc/central-loadbalancer -o json | jq -r '.status.loadBalancer.ingress[0] | .ip // .hostname')
-          kubectl -n stackrox create secret generic access-rhacs --from-literal="username=${ROX_ADMIN_USERNAME}" --from-literal="password=${ROX_ADMIN_PASSWORD}" --from-literal="central_url=https://${CENTRAL_IP}"
-          echo "rox_password=${ROX_ADMIN_PASSWORD}" >> "$GITHUB_OUTPUT"
-          echo "cluster_name=${NAME//./-}" >> "$GITHUB_OUTPUT"
+      - name: Start fake workload
+        env:
+          API_ENDPOINT: localhost:8000
+          ROX_PASSWORD: ${{ steps.launch-central.outputs.rox_password }}
+        run: |
+          #echo "::add-mask::$ROX_PASSWORD"
```

Comment:
I still have not found a way of masking a variable and passing it to another job. This was given as something that might help me https://github.com/orgs/community/discussions/25225

It doesn't give a satisfactory solution. The suggestion there was simply to use base64 encoding. I don't think that is very effective. In some cases github actions automatically masks things preceded by the word password. I am not sure that will help here.

Masked variables can be passed from step to step within the same job, but not from job to job. One solution would simply to put the creation of the secured cluster in the same job. I wanted to put these in separate jobs, because they are separate things. I guess putting it in the same job is not a big deal.